From 4a90ad9754cd2d2f49bfe534985f906dcca4f29e Mon Sep 17 00:00:00 2001
From: Nicholas Devenish <ndevenish@gmail.com>
Date: Mon, 26 Aug 2024 22:04:55 +0100
Subject: [PATCH] Fix bare target_link_libraries

---
 CMakeLists.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index aaca07d2..f9a7d357 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -172,7 +172,7 @@ if (USE_INCLUDED_LIBZIP)
 else()
     find_package(PkgConfig REQUIRED)
     pkg_check_modules(LIBZIP REQUIRED libzip)
-    target_link_libraries(${PROJECT_NAME} ${LIBZIP_LIBRARIES})
+    target_link_libraries(${PROJECT_NAME} PRIVATE ${LIBZIP_LIBRARIES})
 endif()
 
 
@@ -181,7 +181,7 @@ if (USE_INCLUDED_ZLIB)
 else()
     find_package(PkgConfig REQUIRED)
     pkg_check_modules(ZLIB REQUIRED zlib)
-    target_link_libraries(${PROJECT_NAME} ${ZLIB_LIBRARIES})
+    target_link_libraries(${PROJECT_NAME} PRIVATE ${ZLIB_LIBRARIES})
 endif()
 
 set_target_properties(${PROJECT_NAME} PROPERTIES PREFIX "" IMPORT_PREFIX "" )
-- 
2.39.3 (Apple Git-146)

